frontend/tests/artillery/params.json

```diff
@@ -0,0 +1,168 @@
+{
+  "ids": {
+    "local": [
+      1, 2, 3, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21,
+      22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35
+    ],
+    "dev": [],
+    "prod": []
```

Comment:
staging too please! I'll be indexing the load test numbers off staging rather than dev